.vitepress/config.ts

```diff
@@ -95,6 +95,10 @@ export default defineConfig({
       '/platform/webcontainers/': sidebarLinks('main', ['webcontainers']),
       '/enterprise/': sidebarLinks('enterprise', ['enterprise']),
     },
+    chatlio: {
+      allowedRoutes: ["teams\/.*", "enterprise\/.*"],
+      id: process.env.CHATLIO_ID,
```

Comment:
```suggestion
      id: process.env.VITE_CHATLIO_ID,
```

For consistency with the other env variables we use: `VITE_GTM_ID`, `VITE_ALGOLIA_ID`, `VITE_ALGOLIA_KEY`.